src/output_manager/output.rs

```diff
@@ -158,10 +160,6 @@ impl Output {
         let data = self.data();
         data.egui.borrow_mut().run(
             |ctx| {
-                egui::Area::new("main")
-                    .anchor(egui::Align2::LEFT_TOP, (10.0, 10.0))
-                    .show(ctx, |_ui| {});
```

Comment:
This was intentional, as Smithay crashes if egui does not have any elements, was this fixed in the meantime? 
(sorry, I forgot to add a comment about that, and report the issue)

Reply:
Oh, you are right! I never tested for that, I though I left that there.
Ok, add that back for now :)